deployment/config/prometheus.yml

```
@@ -1,9 +1,20 @@
 global:
   scrape_interval: 15s
+  # Enable legacy metric name validation if needed for compatibility
+  metric_name_validation_scheme: legacy
 
 scrape_configs:
   - job_name: "prometheus"
-
+    # Explicitly specify scrape protocol fallback for compatibility
+    scrape_protocols: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)